mariadb/templates/daemonset.yaml.j2

```diff
@@ -0,0 +1,98 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+spec:
+  selector:
+    matchLabels:
+      galera: enabled
+  template:
+    metadata:
+      labels:
+        app: mariadb
+        galera: enabled
+    spec:
+      nodeSelector:
+        {{ deployment.control_node_label }}: enabled
+      # TODO(DTadrzak): it must be removed in the future
+      securityContext:
```

Comment:
MariaDB needs permissions to write to data directory (`/var/lib/mysql-openstack-{{ database.cluster_name }}`). This directory is created as a `hostPath` type of volume - which is always mounted as root. According to https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/volumes.md: 

> The hostPath volume should only be used by effective-root users, and the permissions of paths exposed into containers via hostPath volumes should always be managed by the cluster operator.